client/consensus/manual-seal/src/lib.rs

```diff
@@ -359,14 +358,18 @@ mod tests {
 		let (client, select_chain) = builder.build_with_longest_chain();
 		let client = Arc::new(client);
 		let spawner = sp_core::testing::TaskExecutor::new();
+		let api = api();
```

Comment:
This named local variable here does add much in this context and I suggest to restore it the way it was.

A (bit opinionated) rule of thumb to instance named local variables in two cases:
1. if it is used in multiple places OR
2. if its declaration is so involved to be better readable if is declared as a stand alone var.

Reply:
32a9522